posthog/test/test_dashboard_insight_model.py

```diff
@@ -0,0 +1,29 @@
+from posthog.test.base import BaseTest
+
+
+class TestDashboardInsightModel(BaseTest):
```

Comment:
I don't see the `DashboardInsight` itself. Should this be the table between dashboards and insights? If so, is this the right name for such a table, given it'll contain 1) what to show 2) coordinates of the item on the dashboard, 3) color of the item. The "item" is currently just an insight, but it can also be a block of text, the events list table, or whatever. Should we make a new many-to-many model for every new type of dashboard element?